Git.hub.Demo/Git.hub.Demo.csproj

```diff
@@ -23,6 +23,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Sorry, most of these changes are not my own but things that existed in master but not v0.1.1.0

If some of these changes don't make sense, could someone create a branch on the tag v0.1.1.0 so that I can make a pull request with only my changes?

Or should I revert some of the commits in this pull request?